README.md

```diff
@@ -15,11 +15,22 @@ npm run serve
 npm run build
 ```
 
-### Lints and fixes files
+### Run lint
```

Comment:
Please use [markdown-lint](https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint) to ensure consistent markdown styling 😇